src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs

```diff
@@ -143,6 +143,7 @@ internal static Span ToOtlpSpan(this Activity activity, SdkLimitOptions sdkLimit
                 TraceId = UnsafeByteOperations.UnsafeWrap(traceIdBytes),
                 SpanId = UnsafeByteOperations.UnsafeWrap(spanIdBytes),
                 ParentSpanId = parentSpanIdString,
+                TraceState = activity.TraceStateString ?? string.Empty,
```

Comment:
Curious: which one is preferred, string.Empty or null?

Reply:
I think we should use string.Empty here.
Default behavior (without adding L146) is empty.